app.js

```diff
@@ -1,6 +1,7 @@
 const Express = require('express');
 const BodyParser = require('body-parser');
 const Mongoose = require('mongoose');
+Mongoose.set('useFindAndModify', false);
```

Comment:
It would be good to put this line somewhere below module imports to divide imports from actual app code but that's just preference.

Reply:
This line was removed in the latest commit.